public/push-messaging-sw.js

```diff
@@ -0,0 +1,30 @@
+/* global importScripts firebase FCM_SENDER_ID */
+// Give the service worker access to Firebase Messaging.
+// Note that you can only use Firebase Messaging here, other Firebase libraries
+// are not available in the service worker.
+
+importScripts('/lib/firebase/firebase-app.js');
+importScripts('/lib/firebase/firebase-messaging.js');
+importScripts('/config/sw.js');
```

Comment:
Okay, now I understood why the config was loaded like it was earlier! Ignore my earlier comments about this.

We should probably move all the configs to load like this. :-)